dali/test/python/test_backend_impl_torch_dlpack.py

```diff
@@ -149,7 +149,7 @@ def create_tmp(idx):
 
 
 def check_dlpack_types_cpu(t):
-    arr = torch.tensor([[-0.39, 1.5], [-1.5, 0.33]], device="cpu", dtype=t)
+    arr = torch.tensor([[0.39, 1.5], [1.5, 0.33]], device="cpu", dtype=t)
```

Comment:
The latest torch complains `value cannot be converted to type uint8 without overflow`.